src/routes.tsx

```diff
@@ -8,13 +13,21 @@ import SubmitComplaintPhoto from './pages/SubmitComplain/SubmitComplaitPhoto';
 import SubmitComplaintGeolocation from './pages/SubmitComplain/SubmitComplaintGeolocation';
 import SubmitComplaintConclude from './pages/SubmitComplain/SubmitComplaintConclude';
 import ComplaintDetails from './pages/ComplaintDetails';
+import FirstPage from './pages/FirstPage';
+import RegisterName from './pages/Register/RegisterName';
+import RegisterAdress from './pages/Register/RegisterAdress';
+import RegisterEmail from './pages/Register/RegisterEmail';
 
 const Routes = () => {
 	return (
 		<Router>
 			<Navbar />
 			<Switch>
-				<Route component={Home} path='/' exact />
+				<Route component={FirstPage} path='/' exact />
+				<Route component={RegisterName} path='/register/name' />
+				<Route component={RegisterAdress} path='/register/adress' />
+				<Route component={RegisterEmail} path='/register/email' />
+				<Route component={Home} path='/home' />
```

Comment:
Modificar a rota também nos lugares que a utilizam no código, como no arquivo Drawer.tsx:

							<Link
								className='side-drawer__text'
								to='/'
								onClick={close}
							>
								VER DENÚNCIAS
							</Link>